### ISSUE TITLE
Sileo issue

### ISSUE THREAD

**Issue by stylesbeyond** [LNSSPsd/PodsGrant#26]
I’m getting this now in sileo? What does this mean. Thanks. 

![178FB6BA-5D67-4C3F-B309-67FCAE89B093](https://user-images.githubusercontent.com/66372465/232383024-d56c7b5c-f273-4e2a-99d3-cd7bb8746384.jpeg)
![313424CC-269B-4335-BDF2-028AD68EFED4](https://user-images.githubusercontent.com/66372465/232383031-e39ba9bc-d9a2-47aa-97ca-e7a4a5c9e3d8.png)

**Comment by LNSSPsd** [comment]
It's a mistake on `postinst` script. Try the previous version or ignore it (it will work).

**Comment by stylesbeyond** [comment]
Thanks. I’ve dropped a version. Are we supposed to get the AirPods popup and in findmy playing a sound doesn’t seem to do anything. 15.4.1. Thanks for the quick reply’s and the tweak

**Comment by LNSSPsd** [comment]
The popup is supposed to be working, but the findmy isn't considered currently.

**Comment by stylesbeyond** [comment]
Any updates on this coming? Thanks

**Comment by LNSSPsd** [comment]
> Any updates on this coming? Thanks

Yes, the latest commit should have solved this problem, just wait for the next release.

**Comment by stylesbeyond** [comment]
> > Any updates on this coming? Thanks
> 
> Yes, the latest commit should have solved this problem (pop-up), just wait for the next release.

Thanks. I hope that’s soon

**Comment by LNSSPsd** [comment]
> > > Any updates on this coming? Thanks
> > 
> > 
> > Yes, the latest commit should have solved this problem (pop-up), just wait for the next release.
> 
> Thanks. I hope that’s soon

It has been there, just check it out.
